File: src/pages/exercises/string-citation/_codes/ruby/code/print/citation.print.rb
require_relative "../src/citation"

#Formatting case citation
person = 'Pedro Henrique de Sales Xavier'
puts(Citation.longCitation(name))
puts 'XAVIER; Pedro Henrique de Sales'

#Formatting short form case citation
person = 'Pedro Henrique de Sales Xavier'
puts(Citation.shortCitation(name))
puts 'XAVIER; P. H. D. S.'

File: src/pages/exercises/string-citation/_codes/js/code/citation.print.js
import { citation, compactCitation } from './citation.js';

// Citation Name

// formatting case citation
let person = 'Luiz Carlos Rodrigues Chaves';
console.log(citation(person));
console.log('CHAVES; Luiz Carlos Rodrigues');

// formatting short form case citation
person = 'Luiz Carlos Rodrigues Chaves';
console.log(compactCitation(person));
console.log('CHAVES; L. C. R.');
